things/service.go

```diff
@@ -319,11 +319,19 @@ func (ts *thingsService) ListThings(ctx context.Context, token string, pm PageMe
 	}
 
 	subject := res.GetId()
+	// If the user is admin, fetch all things from database.
 	if err := ts.authorize(ctx, res.GetId(), authoritiesObject, memberRelationKey); err == nil {
-		subject = adminSubject
 		pm.FetchSharedThings = true
+		page, err := ts.things.RetrieveAll(ctx, res.GetEmail(), pm)
+		if err != nil {
+			return Page{}, err
+		}
+		return page, err
+		// subject = adminSubject
```

Comment:
Dead code?

Reply:
Yes, this looks like a dead code. @buraksekili can you please fix this one in a new PR?